mobile/apps/artist/routes.coffee

```diff
@@ -26,6 +34,11 @@ module.exports.index = (req, res, next) ->
     artist.maybeFetchAndSetFeaturedBio()
     artist.fetchArtworks(data: {size: 50})
   ]).then ([artistData, partnerArtists, artworks]) ->
+    res.locals.sd.modal =
```

Comment:
sd properties are traditionally all uppercase. I would also recommend making this one a bit more specific like `APP_BANNER_MODAL` or something.